### DISCUSSION TITLE
Need help in understanding docker function in this app

### DISCUSSION THREAD

**Discussion by abhinav2712** [EddieHubCommunity/LinkFree#5606] [General]
I am a newbie and an aspiring devops engineer and as I was going through the README.md file, I was quite fascinated and curious on how you guys dockerized this app like we can run this like a docker image without having to install node packages !

I request mentors to just give a rough structure and work flow used to make this functionality

**Comment by waellerbe** [comment]
> I am a newbie and an aspiring devops engineer and as I was going through the README.md file, I was quite fascinated and curious on how you guys dockerized this app like we can run this like a docker image without having to install node packages !
> 
> I request mentors to just give a rough structure and work flow used to make this functionality.

Hello @abhinav2712 : While I am not well versed with Docker, I wonder whether you may be able to use Google Colab to experiment with some of the code for Docker projects.
